src/Ims/ImCounters.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace ATDev\RocketChat\Ims;
+
+use ATDev\RocketChat\Common\Counters;
+
+class ImCounters extends Counters
```

Comment:
I don't think we need to call it `ImCounters` since we have the namespace. Just call it `Counters`. It will require full parent objet name with namespace probably for declaration.